APIs/SentinelHub/Batch/Examples.qmd

```diff
@@ -276,7 +276,7 @@ batch_request_id = response.json()['id']
 ### Get information about all your batch processing requests
```

Comment:
Can you add a few descriptive texts after each of these sub-headings as done for Restart a partially processed batch processing request (RESTART) before providing the code snippets?